controller/backup_volume_controller.go

```diff
@@ -314,10 +314,20 @@ func (bvc *BackupVolumeController) reconcile(backupVolumeName string) (err error
 		log.Infof("Found %d backups in the backup target that do not exist in the cluster and need to be pulled", count)
 	}
 	for backupName := range backupsToPull {
+		backupLabelMap := map[string]string{types.GetLonghornLabelKey(types.LonghornLabelVolumeAccessMode): string(longhorn.AccessModeReadWriteOnce)}
+
+		backupURL := backupstore.EncodeBackupURL(backupName, backupVolumeName, backupTargetClient.URL)
+		if backupInfo, err := backupTargetClient.BackupGet(backupURL, backupTargetClient.Credential); err == nil {
```

Comment:
Don't swallow error. If want to continue when encountering error, you can add a warning log to what's happening behind.

Reply:
Yeah, modified, it would have a warning log.